ClassicPress-research/classic-elements
# Broken System Info output

Previous change of class from WordPress_Reporter to ClassicPress_Reporter broke the System Info Screen. This PR fixes that

## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Fix: Broken System Info Screen

## Description
An explanation of what is done in this PR

* Renamed wordpress.php to classicpress.php in includes/settings/system-info/classes
* Changed system-info/main.php to reflect above change

## Test instructions
This PR can be tested by following these steps:

* Build the plugin, install & activate then visit the System Info screen and there should be no errors.

## Quality assurance
